src/index.ts

```diff
@@ -140,7 +140,25 @@ export default function register(vjs: typeof videojs = videojs) {
                     adParameters = display.creative.adParameters || '';
 
                     const mediaUrl = display.media?.fileURL ? new URL(display.media.fileURL) : { hostname: '' };
+                    this.logger.debug('parsing vast for host', mediaUrl.hostname);
```

Comment:
This in here for testing purposes?

Reply:
This is for debugging. The logger class will only log this in debugging mode, I think it will be useful information for anyone implementing this plugin.